standalone-metastore/metastore-server/src/main/sql/mysql/hive-schema-4.0.0.mysql.sql

```diff
@@ -1177,13 +1177,13 @@ CREATE TABLE `SCHEMA_VERSION` (
   `SCHEMA_ID` BIGINT,
   `VERSION` INTEGER NOT NULL,
   `CREATED_AT` BIGINT NOT NULL,
-  `CD_ID` BIGINT, 
+  `CD_ID` BIGINT,
   `STATE` INTEGER NOT NULL,
   `DESCRIPTION` VARCHAR(4000),
   `SCHEMA_TEXT` mediumtext,
   `FINGERPRINT` VARCHAR(256),
   `SCHEMA_VERSION_NAME` VARCHAR(256),
-  `SERDE_ID` bigint, 
+  `SERDE_ID` bigint,
```

Comment:
@HarshitGupta11 : Can you please remove formatting changes for these files? This will help in easier review.